src/tasks/payload/payload_task.c

```diff
@@ -0,0 +1,100 @@
+/**
+ * @author  Marc Aaron Reyes
+ * @date    2025-05-03
+ */
+
+#include "payload_task.h"
+#define MAX_RECEIVED_LEN 1024
+
+void payload_task_init(slate_t *slate)
+{
+    LOG_INFO("Payload task is initializing...");
+
+    LOG_INFO("Initializing UART...");
+    payload_uart_init(slate);
+
+    LOG_INFO("Turning on Payload...");
+    payload_turn_on(slate);
+
+    LOG_INFO("Waiting for Pi to boot up...");
+    sleep_ms(10000);
```

Comment:
For now ok... but think about how to avoid these long duration of sleeps as it'll block execution (and we only have a single thread).

Are there any I'm alive signals from the RPi we can scan/query for each time we try to establish a connection to it?

Also consider something like a stream/file-IO like interface with the RPi, open file --> read/write stuff --> close file. As **init for all tasks only run once immediately after boot**, and we'd like to be able to turn on payload only when we're needing it as a result of commands received from ground station (then turn it off on demand/after some timeout to conserve power).

Reply:
Just to be clear, you don't need to make changes to address this in the current PR, but keep these constraints in mind for a future fix.